packages/sanity/src/core/bundles/util/util.ts

```diff
@@ -1,17 +1,22 @@
 import speakingurl from 'speakingurl'
 
 import {type BundleDocument} from '../../store/bundles/types'
-
-const PUBLISHED_SLUG = 'Published'
+import {isVersionId} from '../../util'
 
 /**
  * @internal
  * @hidden
  */
-export function getBundleSlug(documentId: string): string {
-  if (documentId.indexOf('.') === -1) return PUBLISHED_SLUG
-  const version = documentId.slice(0, documentId.indexOf('.'))
-  return version
+export function getBundleSlug(documentId: string): string | undefined {
+  if (documentId.indexOf('.') === -1) return undefined
+
+  if (isVersionId(documentId)) {
+    // eslint-disable-next-line unused-imports/no-unused-vars, @typescript-eslint/no-unused-vars
+    const [_versionPrefix, bundleSlug] = documentId.split('.')
```

Comment:
nit, but would be able to remove the eslint disable comment by changing to
```suggestion
    const [, bundleSlug] = documentId.split('.')
```

Reply:
Ah I didn't know this was a thing! I'll make this change :)